policy/release/base_image_registries.rego

```diff
@@ -106,12 +101,6 @@ _image_ref_permitted(image_ref) if {
 	image.parse(image_ref).digest in allowed_digests
 }
 
-_base_images contains name if {
-	some _, image in lib.results_named(lib.build_base_images_digests_result_name)
-	some name in split(image.value, "\n")
-	name != ""
-}
-
```

Comment:
For reviewers: This is the functional change right here, everything else is tests and docs.